homeassistant/components/config/config_entries.py

```diff
@@ -261,9 +261,10 @@ class OptionManagerFlowResourceView(FlowManagerResourceView):
 async def system_options_update(hass, connection, msg):
     """Update config entry system options."""
     changes = dict(msg)
-    changes.pop("type")
-    entry_id = changes.pop("entry_id")
+    entry_id = changes["entry_id"]
```

Comment:
You still want to pop here, so you can pass `**changes` to `system_options.update`